packages/replay/src/main.ts

```diff
@@ -245,9 +236,10 @@ function updateStatus(recording: RecordingEntry, status: RecordingEntry["status"
 }
 
 // Convert a recording into a format for listing.
-function listRecording(recording: RecordingEntry) {
+function listRecording(recording: RecordingEntry): ExternalRecordingEntry {
   // Remove properties we only use internally.
-  return { ...recording, buildId: undefined, crashData: undefined };
```

Comment:
Heads up @jaril I think before the code wasn't actually removing `buildId` and `crashData`, just setting their values to undefined. I'm not enforcing with a type (`Omit<RecordingEntry, "buildId" | "crashData">`) that those keys are removed from the object altogether. Not sure if that's what you originally intended though.

Reply:
did you mean @jasonLaster ?

Reply:
Whoops, I misread `git blame`. Thanks @jaril :)